app/src/main/java/org/fossasia/openevent/app/contract/model/EventModel.java

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.app.contract.model;
+
+import org.fossasia.openevent.app.data.models.Event;
+import org.fossasia.openevent.app.data.models.User;
+
+import java.util.List;
+
+import io.reactivex.Observable;
+
+public interface EventModel {
```

Comment:
@iamareebjamal please make changes in the interfaces(created in this commit) names as we discussed earlier.